tests/pytests/test_vecsim.py

```diff
@@ -320,7 +320,7 @@ def test_create():
     dummy_val = 'dummy_supplement'
 
     # Test for INT32, INT64 as well when support for these types is added.
-    for data_type in VECSIM_DATA_TYPES:
+    for data_type in VECSIM_DATA_TYPES + ['INT8', 'UINT8']:
```

Comment:
Is that the only test we want to apply to int8/uint8? I believe we should have it for a few more (what is the status with FP16?)

Reply:
Most of our tests are not generic enough to run with int types. I also don't think we need to have too many tests for the new types on RediSearch, as long as we cover all the expected functionality (create, index, JSON index, search)
This test covers most of the required functionalities, so It's a good place to test the new types.